playkit/src/main/java/com/kaltura/playkit/plugins/ads/ima/IMAPlugin.java

```diff
@@ -590,11 +590,11 @@ public void onAdEvent(com.google.ads.interactivemedia.v3.api.AdEvent adEvent) {
                     appInBackgroundDuringAdLoad = true;
                     adsManager.pause();
                 } else {
-                    messageBus.post(new AdEvent(AdEvent.Type.LOADED));
                     if (adPlaybackCancelled) {
```

Comment:
I just moved the Event to be sent if it is really continue with the ad flow 
so now event is not sent if ad break is canceled